src/main/java/networkbook/ui/CommandBox.java

```diff
@@ -30,6 +51,43 @@ public CommandBox(CommandExecutor commandExecutor) {
         this.commandExecutor = commandExecutor;
         // calls #setStyleToDefault() whenever there is a change to the text of the command box.
         commandTextField.textProperty().addListener((unused1, unused2, unused3) -> setStyleToDefault());
+        commandHistory = new ArrayList<>();
+        pointer = 0;
+        setCommandBoxShortcuts();
+    }
+
+    private void setCommandBoxShortcuts() {
+        commandTextField.setOnKeyPressed(new EventHandler<KeyEvent>() {
+            @Override
+            public void handle(KeyEvent event) {
+                if (SHORTCUT_FIND.match(event)) {
+                    autoFillCommandIfEmpty("find ");
+                    event.consume();
+                } else if (SHORTCUT_CREATE.match(event)) {
+                    autoFillCommandIfEmpty("create ");
+                    event.consume();
+                } else if (SHORTCUT_EDIT.match(event)) {
+                    autoFillCommandIfEmpty("edit ");
+                    event.consume();
+                } else if (SHORTCUT_UNDO.match(event)) {
+                    autoFillCommandIfEmpty("undo");
+                    event.consume();
+                } else if (SHORTCUT_REDO.match(event)) {
+                    autoFillCommandIfEmpty("redo");
+                    event.consume();
+                } else if (noModifier(event) && event.getCode() == SHORTCUT_UP) {
+                    navigateCommandHistory(true);
+                    event.consume();
+                } else if (noModifier(event) && event.getCode() == SHORTCUT_DOWN) {
+                    navigateCommandHistory(false);
+                    event.consume();
+                }
```

Comment:
Would it be better to use `COMMAND_WORD` field in the respective command,
rather than hardcoding them here?

Reply:
Yep okay will replace these hard-coded texts here and also in the test cases.